includes/admin/class-rop-admin.php

```diff
@@ -314,7 +314,7 @@ public function enqueue_scripts() {
 		$array_nonce['added_networks']          = $added_networks;
 
 		$admin_url = get_admin_url( get_current_blog_id(), 'admin.php?page=TweetOldPost' );
-		$token     = get_option( ROP_INSTALL_TOKEN_OPTION );
+		$token     = get_option( 'ROP_INSTALL_TOKEN_OPTION' );
```

Comment:
here you are using 
```'ROP_INSTALL_TOKEN_OPTION'```

but https://github.com/Codeinwp/tweet-old-post/pull/620/files/92b2578ed47da613dfad485e4ecfce3a94ae75c8..66112c42d102918abe04947ef4aed965f3402674#diff-1e90dad9be0c02bd5bf80125d39eb637R67 you are using it as a constant. 

Should this be ```ROP_APP_TOKEN_OPTION``` instead?

Can you check?

Reply:
Thanks, missed that